src/nspawn/nspawn-register.c

```diff
@@ -44,7 +44,12 @@ static int append_machine_properties(
                                    * do not permit the
                                    * container to ever create
                                    * these device nodes. */
-                                  "char-pts", "rw");
+                                  "char-pts", "rw",
+                                  /* Allow the conntainer to
+                                   * use FUSE, if
+                                   * copy_devnodes() included
+                                   * it. */
+                                  "/dev/fuse", "rw");
```

Comment:
This should be conditional on having the new fuse version too

Reply:
I just realized that I forgot to edit `systemd-nspawn@.service.in` as well, where I don't believe that there's a way for it to be conditional.  I guess that's an OK compromise the same way that it including `/dev/loop-control` and others is an OK compromise.

Just for my own understanding of how DevicePolicy works: What's the harm in it being unconditional here? If `copy_devnodes()` didn't create it, since the policy is `rw` and not `rwm` doesn't that effectively mean it can't ever exist and therefore can't be used?

Reply:
Done. This ended up being sorta gross because I had to pass `enable_fuse` back over the socket from the outer-child to the parent, so it touched a comparatively large amount of code.

Reply:
I would still appreciate an answer to my question though.